file/types.go

```diff
@@ -456,6 +456,7 @@ type FConsumer struct {
 	BasicAuths    []*kong.BasicAuth        `json:"basicauth_credentials,omitempty" yaml:"basicauth_credentials,omitempty"`
 	Oauth2Creds   []*kong.Oauth2Credential `json:"oauth2_credentials,omitempty" yaml:"oauth2_credentials,omitempty"`
 	ACLGroups     []*kong.ACLGroup         `json:"acls,omitempty" yaml:"acls,omitempty"`
+	MTLSAuths     []*kong.MTLSAuth         `json:"mtlsauth_credentials,omitempty" yaml:"mtlsauth_credentials,omitempty"`
```

Comment:
Should the json field name be 'mtls_auth_credentials' instead of  'mtlsauth_credentials' as it is the name of the table in Kong's database ?

Reply:
Yeah--that's annoying, we broke with the table naming convention used for `basicauth_credentials` and `keyauth_credentials` there, but decK should reflect it as it is.